### DISCUSSION TITLE
Outdated or wrong Chinese localization

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14100] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

1. missionname.killcrawlerswarm1 / missionname.killcrawlerswarm2 and all related Chinese translations are outdated. It refers specifically to Crawler Swarm in current Chinese translation instead of an ambiguous Swarm or [monster] variant
2. Unknown output <deconstructor.unknownitemsoutput> is translated into unknow outpost
3. Duration of respawn transport <serversettingsrespawnduration> is translated into time required to respawn
4. (Submarine selection) Manual <manual> is translated into manual book
5. Lowers the game's volume when players are speaking <voipattenuationtooltip> is translated into Lowers volume when players are speaking which is confusing
6. Cutoff prevention <cutoffprevention> is translated into background noises prevention and translation of <cutoffpreventiontooltip> is incorrect too
7. Disrupt the mining efforts by swapping oxygen tanks in 50% of the plasma cutters for welding fuel tanks. <EventText.miningsabotage.objective1> is translated into swapping welding tanks in 50% of the welding tools for oxygen tanks
8. Stackable materials can have +[amount] items per stack in your inventory <talentdescription.mule> is translated into Stackable items

### Reproduction steps

check corresponding text files

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I've created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/14106), we'll let the localization team know about these issues.
